src/inventorymanager.cpp

```diff
@@ -248,6 +248,12 @@ void IMoveAction::apply(InventoryManager *mgr, ServerActiveObject *player, IGame
 		to_inv.type == InventoryLocation::PLAYER &&
 		from_inv.name == to_inv.name);
 
+	/*
+		Fix Drag & Drop
+	*/
+
+	ItemStack replaced = list_to->getItem(to_i);
+	bool fix_replacements = !replaced.empty();
```

Comment:
bad variable name

Reply:
I know. Would `dst_item` and `dst_item_replaced`(for ItemStack and boolean respectively) be better?

Reply:
I would name it `is_being_replaced`, not too sure about your suggestions but it's probably fine